src/core/network/createPayload.js

```diff
@@ -30,6 +30,12 @@ export default () => {
       content.events.push(event);
     },
     mergeMeta: createMerger(content, "meta"),
+    get expectsResponse() {
+      return (
+        isNonEmptyArray(content.events) &&
+        find(content.events, event => event.expectsResponse) !== undefined
```

Comment:
It might be worth implementing a `some` utility that basically does this and matches the [Array.some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) method. Whatever you'd like to do is fine with me though.

Reply:
Actually, Array.some is supported in all our target browsers. You can just use it.